README.md

```diff
@@ -1,57 +1,269 @@
+
+A className to apply to the content container of each item.
+
+#### contentClassName
+
+Type: `string` | Required: `false`
+
+A className to apply to the content of each item.
+
+#### iconClassName
+
+Type: `string` | Required: `false`
+
+A className to apply to the icon container of each item.
+
+#### onItemToggling
+
+Type: `function` | Required: `false` | Arguments: `index: Number, isActive: Object`
+
+Invoked before an item is opened or closed.
+
+#### onItemToggle
+
+Type: `function` | Required: `false` | Arguments: `index: Number, isActive: Object`
+
+Invoked after an item is opened or closed.
+
+#### scrollIntoView
+
+Type: `boolean` | Required: `false` | Default: `false`
+
+Whether or not the content of an item should be scrolled to be visible.
+
+### AccordionItem Props
+
+#### title
+
+Type: `any` | Required: `true`
+
+The content of the title.
+
+#### children
+
+Type: `node` or `function` | Required: `false`
+
+The content of the item.  
+Example with function:
 
-The `{package-api-prop-example}` has the following shape:
 ```js
-{package-api-prop-example}: PropTypes.shape({
-    foo: PropTypes.string,
-    bar: PropTypes.arrayOf(PropTypes.object),
-}).isRequired,
+<AccordionItem { ...props }>
+    { ({ isActive }) => isActive ? 'Open' : 'Closed' }
+</AccordionItem>
+```
+
+#### animationDuration
+
+Type: `number` | Required: `false` | Default: `300`
+
+Duration of the animation when the item is opened or closed.
+
+#### animationEasing
+
+Type: `string` | Required: `false` | Default: `ease-in-out`
+
+Type of animation easing when the item is opened or closed.
+
+#### itemContainerClassName
+
+Type: `string` | Required: `false`
+
+A className to apply to the item container.
+
+#### headingContainerClassName
+
+Type: `string` | Required: `false`
+
+A className to apply to the heading container.
+
+#### titleContainerClassName
+
+Type: `string` | Required: `false`
+
+A className to apply to the title.
+
+#### contentContainerClassName
+
+Type: `string` | Required: `false`
+
+A className to apply to the content container.
+
+#### contentClassName
+
+Type: `string` | Required: `false`
+
+A className to apply to the content.
+
+#### iconClassName
+
+Type: `string` | Required: `false`
+
+A className to apply to the icon container.
+
+#### icon
+
+Type: `any` | Required: `false`
+
+The content of the icon.
+
+#### index
+
+Type: `number` | Required: `false`
+
+The index of the item.
+
+#### isActive
+
+Type: `boolean` | Required: `false` | Default: `false`
+
+Whether or not the item is active.
+
+#### onItemToggle
+
+Type: `function` | Required: `false` | Arguments: `index: Number, isOpen: Boolean`
+
+Invoked after an item is opened or closed.
+
+#### scrollIntoView
+
+Type: `boolean` | Required: `false`
+
+Whether or not the content of an item should be scrolled to be visible.
+
+### toggleItemActive
+
+Type: `function` | Required: `false` | Arguments: `index: Number`
+
+Function to handle the indices of the open items.
+
```

Comment:
It seems like we have the proptypes of the AccordionItem here, but I don't think this should be the case. Some props should not be available for external use since they are being passed by the parent itself (Accordion).

For example, `isActive` and `toggleItemActive`.

Please, make sure we only transcribe the props we should actually accept.

Reply:
I agree. I will remove all the props controlled by the `Accordion`.